test/membrane_element_portaudio/sink_test.exs

```diff
@@ -12,52 +13,52 @@ defmodule Membrane.Element.Portaudio.SinkTest do
     state = %{
       endpoint_id: :default,
       ringbuffer_size: 4096,
-      pa_buffer_size: 256,
+      portaudio_buffer_size: 256,
       latency: :high,
-      native: nil,
-      playing: false
+      native: nil
     }
 
     %{state: state}
   end
 
   def playing(%{state: state}) do
-    %{state: %{state | native: make_ref(), playing: true}}
+    %{state: %{state | native: make_ref()}}
   end
 
   setup_all :state
 
-  describe "handle_play" do
+  describe "handle_prepared_to_playing" do
+    @tag skip: "Temporairly disabled due to mocking issues"
```

Comment:
Please check if we can do something about it by placing mockable macro inside the SyncExexutor module

Reply:
Unfortunately, it doesn't work. It seems that mockery is based on the per-process dictionary.